lib/renderer.js

```diff
@@ -12,9 +12,17 @@ module.exports = function(data, options) {
     this.log.warn(`Deprecated config detected. Please use\n\nmarkdown:\n  preset: ${markdown.preset}\n\nSee https://github.com/hexojs/hexo-renderer-markdown-it#options`);
   }
 
-  const { preset, render, plugins, anchors } = markdown;
+  const { preset, render, enable_rules, disable_rules, plugins, anchors } = markdown;
   let parser = new MdIt(preset, render);
 
+  if (enable_rules) {
+    parser.enable(enable_rules);
```

Comment:
Is `parser.enable` and `parser.disable` robust enough? Should we casts array (`Array.isArray(arr) ? arr : [arr]`)?

Reply:
Should be, the [docs](https://github.com/markdown-it/markdown-it#manage-rules) give examples in both string and array.

Reply:
I updated unit test to include both string and array.